x-pack/plugins/cloud_security_posture/public/components/csp_counter_card.tsx

```diff
@@ -7,48 +7,43 @@
 
 import React from 'react';
 import { css } from '@emotion/react';
-import { EuiCard, EuiIcon, EuiStat, useEuiTheme } from '@elastic/eui';
-import type { EuiStatProps, EuiCardProps } from '@elastic/eui';
+import { EuiCard, EuiIcon, EuiText, EuiTitle, useEuiTheme } from '@elastic/eui';
+import type { EuiTextProps, EuiCardProps } from '@elastic/eui';
 
-export type CspCounterCardProps = Pick<EuiCardProps, 'onClick' | 'id'> &
-  Pick<EuiStatProps, 'title' | 'description' | 'titleColor' | 'isLoading'>;
+export type CspCounterCardProps = Pick<EuiCardProps, 'onClick' | 'id' | 'title' | 'description'> & {
+  descriptionColor?: EuiTextProps['color'];
+};
 
-export const CspCounterCard = ({ counter }: { counter: CspCounterCardProps }) => {
+export const CspCounterCard = (counter: CspCounterCardProps) => {
```

Comment:
Consider de-structuring props in the arguments: `({ title, onClick, id, description, descriptionColor })`